smf/manifests/buckets-mdapi-setup.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0"?>
+<!DOCTYPE service_bundle SYSTEM "/usr/share/lib/xml/dtd/service_bundle.dtd.1">
+<!--
+    This Source Code Form is subject to the terms of the Mozilla Public
+    License, v. 2.0. If a copy of the MPL was not distributed with this
+    file, You can obtain one at http://mozilla.org/MPL/2.0/.
+-->
+
+<!--
+    Copyright 2020 Joyent, Inc.
+-->
+
+<service_bundle type="manifest" name="buckets-mdapi-setup">
+    <service name="manta/application/buckets-mdapi-setup" type="service" version="1">
+
+        <create_default_instance enabled="true" />
+
+        <dependency name="network" grouping="require_all" restart_on="error" type="service">
+            <service_fmri value="svc:/network/physical" />
+        </dependency>
+        <dependency name="filesystem" grouping="require_all" restart_on="error" type="service">
+            <service_fmri value="svc:/system/filesystem/local" />
+        </dependency>
+        <dependency name="mdata" grouping="require_all" restart_on="none" type="service">
+            <service_fmri value="svc:/smartdc/mdata:execute" />
+        </dependency>
+        <dependency name="config-agent" grouping="optional_all" restart_on="none" type="service">
+            <service_fmri value="svc:/smartdc/application/config-agent" />
+        </dependency>
+
+        <exec_method type="method" name="start" timeout_seconds="0"
```

Comment:
Just for my understanding, the 0 second timeout means "forever"?

Reply:
Yup

Reply:
To the degree that these docs are authoritative:

Limiting the Time To Wait for Self-Assembly To Complete

> The example SMF service manifest shown above defines timeout_seconds='0' for the start method. This means that SMF will wait indefinitely for self-assembly to complete.

from https://docs.oracle.com/cd/E26502_01/html/E21383/smfdelivery.html